easybuild/easyconfigs/m/MAGeCK/MAGeCK-0.5.9.5-gfbf-2022b.eb

```diff
@@ -0,0 +1,43 @@
+# This easyconfig was created by the BEAR Software team at the University of Birmingham.
+# Updated: Denis Kristak (Inuits)
+easyblock = 'PythonPackage'
+
+name = 'MAGeCK'
+version = '0.5.9.5'
+
+homepage = "https://sourceforge.net/p/mageck/wiki/Home/"
+description = """Model-based Analysis of Genome-wide CRISPR-Cas9 Knockout (MAGeCK) is a computational tool to identify
+ important genes from the recent genome-scale CRISPR-Cas9 knockout screens (or GeCKO) technology. MAGeCK is developed
+ by Wei Li and Han Xu from Dr. Xiaole Shirley Liu's lab at Dana-Farber Cancer Institute, and is being actively updated
+ by Wei Li lab from Children's National Medical Center."""
+
+toolchain = {'name': 'gfbf', 'version': '2022b'}
+
+dependencies = [
+    ('Python', '3.10.8'),
+    ('SciPy-bundle', '2023.02'),
+]
+
```

Comment:
The `preinstallopts = "rm bin/{mageckGSEA,RRA} && "` is no longer needed for this version indeed, since it doesn't include those pre-built binaries anymore (and since it's `rm` and not `rm -f`, it actually breaks the install)